docs/csharp/language-reference/compiler-messages/cs0116.md

```diff
@@ -9,9 +9,11 @@ ms.assetid: 4cb137b5-ec29-4c1a-adde-9f8424cb9496
 ---
 # Compiler Error CS0116
 
-A namespace does not directly contain members such as fields or methods  
+A namespace cannot directly contain members such as fields or methods  
```

Comment:
Updated this line based on this code:
https://github.com/dotnet/roslyn/blob/1fbd48157d9196ad227f120b731d3f80b1b162de/src/Compilers/CSharp/Portable/CSharpResources.resx#L652